server/itx/server.go

```diff
@@ -232,20 +230,6 @@ func StartServer(ctx context.Context, svr *Server, probeSvr *probe.Server, cfg c
 		}()
 	}
 
-	// check disk space
-	m, err := disk.NewMonitor(cfg.Chain.ChainDBPath, 3*time.Minute)
-	if err != nil {
-		log.L().Panic("Failed to create monitor disk space.", zap.Error(err))
-	}
-	if err = m.Start(ctx); err != nil {
-		log.L().Panic("Failed to start monitor disk space.", zap.Error(err))
-	}
-	defer func() {
-		if err := m.Stop(ctx); err != nil {
-			log.L().Panic("Failed to stop monitor disk space.", zap.Error(err))
-		}
-	}()
-
```

Comment:
looks like you are reverting an earlier commit? can you open another PR to just revert that commit?

Reply:
Ok. Split this pr into https://github.com/iotexproject/iotex-core/pull/3685 and https://github.com/iotexproject/iotex-core/pull/3686